cfgov/jinja2/v1/_includes/molecules/form-field-with-button.html

```diff
@@ -0,0 +1,49 @@
+{# ==========================================================================
+
+   form-field-with-button.render()
+
+   ==========================================================================
+
+   Description:
+
+   Creates a form field with text when given:
+
+   settings:  An object used to customize the markup.
+
+   settings.btn_text:          The text for the submit button.
+
+   settings.field.attributes:  Additional attributes to add the input.
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/48296b23d19a0a10a8fb65bbda2bda9340796333